tests_end_to_end/test_dockerserver.py

```diff
@@ -0,0 +1,66 @@
+import requests
+from requests.sessions import HTTPAdapter
+from urllib3 import Retry
+
+from demisto_sdk.commands.common.MDXServer import (DEMISTO_DEPS_DOCKER_NAME,
+                                                   start_docker_MDX_server)
+from demisto_sdk.commands.lint.docker_helper import init_global_docker_client
+
+
+def assert_successful_mdx_call():
+    session = requests.Session()
+    retry = Retry(total=2)
+    adapter = HTTPAdapter(max_retries=retry)
+    session.mount('http://', adapter)
+    response = session.request(
+        'POST',
+        'http://docker:6161',
+        data='## Hello',
+        timeout=20
+    )
+    assert response.status_code == 200
+
+
+def assert_not_successful_mdx_call():
+    session = requests.Session()
+    retry = Retry(total=2)
+    adapter = HTTPAdapter(max_retries=retry)
+    session.mount('http://', adapter)
+    response = session.request(
+        'POST',
+        'http://docker:6161',
+        data='<div> Hello',
+        timeout=20
+    )
+    assert response.status_code == 500
+
+
+def container_is_up():
+    return any(container.name == DEMISTO_DEPS_DOCKER_NAME
+               for container in init_global_docker_client().containers.list())
+
+
+def test_is_file_not_valid():
+    with start_docker_MDX_server() as boolean:
```

Comment:
Are we actually creating a server while testing? Shouldn't this be mocked?

Reply:
this is under end_to_end, running once during nightly, so I dont think there would be an issue